client/index.html

```diff
@@ -1,14 +1,62 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <title>Eventful</title>
-    <link rel="stylesheet" media="screen" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">
-</head>
-<body>
-  <div id="content"></div>
-  <!-- build:js -->
-  <script src="js/App.js"></script>
-  <!-- endbuild -->
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+    <meta name="description" content="Catering event staffing made simple.">
+    <meta name="author" content="">
+    <!-- <link rel="icon" href="../../favicon.ico"> -->
+
+    <title>Eventfull</title>
+
+    <!-- Preload Fonts -->
+    <link href='http://fonts.googleapis.com/css?family=Alegreya+Sans:300,700,800italic' rel='stylesheet' type='text/css'>
+
+    <!-- Bootstrap core CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">
+
+    <!-- Custom styles for Eventfull -->
+    <link href="style/eventfull.css" rel="stylesheet">
+
+    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+    <!--[if lt IE 9]>
+      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
+  </head>
+
+  <body>
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-sm-3 col-md-2 sidebar">
+          <div class="logo-offset">
+            <img src="./style/EventfullLogo@2x.png" width="160px" alt="Eventfull">
+          </div>
+          <ul class="nav nav-sidebar">
```

Comment:
The styles associated with the sidebar are slightly more complicated than with typical components. The thought here, and with any more difficult CSS, is to build a static version and then abstract the HTML structure and styles into React components. Since we haven't determined exactly how we want to utilize the sidebar, I thought it would be helpful to throw up a static placeholder and address later.

Having said all of that, this is a great suggestion and I hadn't really thought about how easy it would be to abstract at this point. Good eye Miyagi, either way works for me.